Client/Ecosia/UI/MultiplyImpact/MultiplyImpact.swift

```diff
@@ -570,24 +564,29 @@ final class MultiplyImpact: UIViewController, Themeable {
     }
     
     private var inviteMessage: String? {
-        guard let link = inviteLink else { return nil }
+        guard let inviteDeepLink, let inviteUrl = URL(string: inviteDeepLink) else { return nil }
         
         return """
-\(String(format: .localized(.iThinkYouWillLikeThis), activeUsers))
+\(String(format: .localized(.messageMentioningActiveUsers), activeUsers))
 
-\(String.localized(.downloadTheApp))
-https://ecosia.co/install-ios
+\(Referrals.sharingLinkRoot)\(inviteUrl.lastPathComponent)
```

Comment:
Why not use `inviteLink` in here? Isn't it the same?

Reply:
No reason not to 💪 . Did not update that when added `private var inviteLink: String?` utilized to update the link in the modal. Thanks!